server/src/main/java/io/crate/metadata/FulltextAnalyzerResolver.java

```diff
@@ -192,9 +193,10 @@ public boolean hasBuiltInCharFilter(String name) {
     }
 
     public Set<String> getBuiltInCharFilters() {
-        return new ImmutableSet.Builder<String>().addAll(EXTENDED_BUILTIN_CHAR_FILTERS)
```

Comment:
Is the `ImmutableSet` import then still used? does it make sense to adjust any other possible usages right now and get rid of it here?

Reply:
There were 2 more places - not sure why I missed those.

Replacing them now.